### PR TITLE
Generate Readme.md

### LINKED ISSUE #1757: Create README.md file for `flow init`
### Issue To Be Solved

Currently, `flow init` says to the user to look at README.md for more information.  However, this file does not exist.

### (Optional): Suggest A Solution

This file should serve 2 purposes:
1. Summary of files generated by Flow CLI & dependencies installed
2. Instructions of how to use project (test, deploy, install more dependencies) + links to docs

### PR DESCRIPTION
**Pull request by jribbink**
Closes #1757 

## Description

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: internal/super/generator/fixtures/README_no_deps.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+## 👋 Welcome Flow Developer!
+
+Welcome to your new Flow project. This project is a starting point for you to develop smart contracts on the Flow Blockchain. It comes with example contracts, scripts, transactions, and tests to help you get started.
+
+## 🔨 Getting started
+
+Here are some essential resources to help you hit the ground running:
+
+- **[Flow Documentation](https://developers.onflow.org/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```
Comment (gregsantos) [inline internal/super/generator/fixtures/README_no_deps.md:9]:
```suggestion
- **[Flow Documentation](https://developers.flow.com/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```

File: internal/super/generator/fixtures/README_with_deps.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+## 👋 Welcome Flow Developer!
+
+Welcome to your new Flow project. This project is a starting point for you to develop smart contracts on the Flow Blockchain. It comes with example contracts, scripts, transactions, and tests to help you get started.
+
+## 🔨 Getting started
+
+Here are some essential resources to help you hit the ground running:
+
+- **[Flow Documentation](https://developers.onflow.org/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```
Comment (gregsantos) [inline internal/super/generator/fixtures/README_with_deps.md:9]:
```suggestion
- **[Flow Documentation](https://developers.flow.com/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```

File: internal/super/generator/templates/README.md.tmpl [hunk 3]
```diff
@@ -0,0 +1,166 @@
+## 👋 Welcome Flow Developer!
+
+Welcome to your new Flow project. This project is a starting point for you to develop smart contracts on the Flow Blockchain. It comes with example contracts, scripts, transactions, and tests to help you get started.
+
+## 🔨 Getting started
+
+Here are some essential resources to help you hit the ground running:
+
+- **[Flow Documentation](https://developers.onflow.org/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```
Comment (gregsantos) [inline internal/super/generator/templates/README.md.tmpl:9]:
```suggestion
- **[Flow Documentation](https://developers.flow.com/)** - The official Flow Documentation is a great starting point to start learning about about [building](https://developers.flow.com/build/flow) on Flow.
```

**Review by gregsantos — APPROVED** [review]
lg2G. couple link suggestions

**Review by chasefleming — APPROVED** [review]
This is great. Only nit is mixed use of title case on headers. Great work!

**Pull request merged**